islandora_transforms/BPL_MODS_to_solr.xslt

```diff
@@ -139,6 +154,28 @@
     <xsl:apply-templates mode="slurp_for_bpl_subject_descendants" select="."/>
   </xsl:template>
 
+  <!-- Concatenate filing-suffix and call-number with space. -->
+  <xsl:template mode="slurp_for_bpl" match="mods:mods">
+    <xsl:variable name="content">
+      <xsl:choose>
```

Comment:
Is this the expected behaviour? The description in the ticket is a little unclear; I'm curious if:

* The 'suffix' should be coming before the call number proper
* The field should be populated with a suffix only if no call number exists.

Regardless, the "and not()"s in the second half should not be necessary, as `xsl:choose` is a first-match-type kinda deal